cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/FindTagsActionHandler.java

```diff
@@ -33,4 +34,24 @@ public Object doGet(Map<String, String> parameters) {
     public Class<IdList> inClass() {
         return IdList.class;
     }
+
+    @Override
+    public String getRoute() {
+        return "find/tags/all";
+    }
+
+    @Override
+    public String getDescription(RestHttpMethod method) {
+        return "Find all tags";
+    }
+
+    @Override
+    public Class<Tag> outClass() {
```

Comment:
I suggest to have a dedicated result object for this operation. In particular, the return type of (see line 30) this operation is `List<Tag>`, which is only observable by checking line 55